groups/live-windows-security-groups/locals.tf

```diff
@@ -0,0 +1,128 @@
+# ------------------------------------------------------------------------
+# Locals
+# ------------------------------------------------------------------------
+locals {
+
+  # ------------------------------------------------------------------------------
+  # Test/Dev Generate Server Security Group Variables
+  # ------------------------------------------------------------------------------
+
+  # Test Dev Generate RDP port CIDR blocks
+  test_dev_generate_rdp_cidr_block = [
+    "10.172.116.224/32",
+    "10.172.23.39/32"
+  ]
+
+  # Test Dev Generate 49155 port CIDR blocks
+  test_dev_generate_49155_cidr_block = [
+    "172.16.101.82/32",
+    "172.16.202.91/32"
+  ]
+
+  # Test Dev Generate 135 port CIDR blocks
+  test_dev_generate_135_cidr_block = [
+    "172.16.101.82/32",
+    "172.16.202.91/32",
+    "172.16.200.200/32"
+  ]
+
+  # ------------------------------------------------------------------------------
+  # Smart Vault 1 Server Security Group Variables
+  # ------------------------------------------------------------------------------
+
+  smart_vault_1_http_cidr_block = [
+    "10.172.116.0/24",
+    "10.172.117.0/24",
+    "10.172.118.0/24",
+    "10.172.119.0/24",
+    "10.172.20.0/24",
+    "10.172.21.0/24",
+    "10.172.22.0/24",
+    "10.172.23.0/24",
```

Comment:
I believe you could lower this again to `10.172.20.0/22` which should cover: `10.172.20.1 - 10.172.23.254`

Same for the `10.172.116.0/24` -> `10.172.116.0/22`

Reply:
smart_vault_1_http_cidr_block = [
    "10.172.116.0/22",
    "10.172.20.0/22",
    "172.16.200.200/32",
    "172.18.96.13/32",
    "172.19.28.13/32",
    "172.19.68.0/24",
    "172.23.0.15/32"
  ]